charts/spire/values.yaml

```diff
@@ -41,10 +41,30 @@ spire-agent:
   enabled: true
   nameOverride: agent
 
+upstream:
+  # -- enable upstream csi driver and agent for use with nested spire.
+  enabled: false
```

Comment:
I think the enabled flag should be moved to the `upstream-spire-agent` and `upstream-csi-driver` sections, or move the csi driver and agent under `upstream:`

```
upstream:
   enabled:false
   spire-agent: ...
   spiffe-csi-driver: ...
```